lemur/common/utils.py

```diff
@@ -128,11 +161,11 @@ def generate_private_key(key_type):
     """
 
     _CURVE_TYPES = {
-        "ECCPRIME192V1": ec.SECP192R1(),
-        "ECCPRIME256V1": ec.SECP256R1(),
-        "ECCSECP192R1": ec.SECP192R1(),
+        "ECCPRIME192V1": ec.SECP192R1(),  # duplicate
+        "ECCPRIME256V1": ec.SECP256R1(),  # duplicate
+        "ECCSECP192R1": ec.SECP192R1(),  # duplicate
         "ECCSECP224R1": ec.SECP224R1(),
-        "ECCSECP256R1": ec.SECP256R1(),
+        "ECCSECP256R1": ec.SECP256R1(),  # duplicate
```

Comment:
Can the comment `# duplicate` be removed now that duplicates are removed?

Reply:
the duplicates are not removed yet. 
I think in a separate PR, we should churn all not support curves, based on what public CAs typically support, for instance: 

https://github.com/letsencrypt/boulder/blob/74c313968063af51c63580e90a2b3bf342bc343b/cmd/ceremony/ecdsa.go#L22-L28